test/test_topology.py

```diff
@@ -540,8 +541,8 @@ def test_wire_version(self):
                 HelloCompat.LEGACY_CMD: True,
                 "setName": "rs",
                 "hosts": ["a"],
-                "minWireVersion": 21,
-                "maxWireVersion": 22,
+                "minWireVersion": max_version + 1,
+                "maxWireVersion": max_version + 2,
```

Comment:
I understand your motivation here but I think it's good to keep this test separate from `common.MAX_SUPPORTED_WIRE_VERSION`, that way when we change MAX_SUPPORTED_WIRE_VERSION we know it has an effect.

Reply:
Done